tests/Unit/Helpers/IO/Observers/MockH5.cpp

```diff
@@ -0,0 +1,77 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Helpers/IO/Observers/MockH5.hpp"
+
+#include <cstddef>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+
+namespace observers::test_helpers {
```

Comment:
`TestHelpers::observers` (I forget what the capitalization in `src` is, but maybe match that?)